src/components/ui/WizardFooter.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import {
+  ButtonVariant,
+  Spinner,
+  Stack,
+  StackItem,
+  Text,
+  TextVariants,
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem,
+} from '@patternfly/react-core';
+import { ToolbarButton } from './Toolbar';
+
+export type WizardFooterGenericProps = {
+  onNext?: () => void;
+  onBack?: () => void;
+  onCancel?: () => void;
+  isNextDisabled?: boolean;
+  isBackDisabled?: boolean;
+  isSubmitting?: boolean;
+  submittingText?: string;
+};
+
+type WizardFooterProps = WizardFooterGenericProps & {
+  alerts?: React.ReactNode;
+  errors?: React.ReactNode;
+  leftExtraActions?: React.ReactNode;
+};
+
+const WizardFooter: React.FC<WizardFooterProps> = ({
+  alerts,
+  errors,
+
+  onNext,
+  onBack,
+  onCancel,
+  isNextDisabled,
+  isBackDisabled,
+  leftExtraActions,
+  isSubmitting,
+  submittingText = 'Saving changes...',
+}) => {
+  const rightExtraActions = isSubmitting ? (
```

Comment:
Should it be possible to pass `rightExtraActions` as a prop? If not it could be included directly in the return jsx instead of having it in a variable.

Reply:
Done. Both recent usecases share the same rightExtraCtions, so there is no need to extract it.